server/area_manager.py

```diff
@@ -500,6 +500,9 @@ def change_lights(self, new_lights, initiator=None):
                 New light status
             initiator: server.ClientManager.Client, optional
                 Client who triggered the light status change.
+            area: server.ClientManager.Client.area, optional
+                Broadcasts light change messages to chosen area. Used if
```

Comment:
You should describe better here that if you set `area` here, the initiator does not get any notifications. Someone who just reads the docstring wouldn't know this.